pandas/tests/groupby/test_categorical.py

```diff
@@ -81,6 +81,7 @@ def get_stats(group):
     assert result.index.names[0] == "C"
 
 
+@pytest.mark.filterwarnings("ignore:.*value of numeric_only.*:FutureWarning")
```

Comment:
can you explictily test these rather than filtering (alt pass `numeric_only=False`) as needed